packages/web-components/fast-components/src/anchored-region/anchored-region.definition.ts

```diff
@@ -17,21 +17,27 @@ export const fastAnchoredRegionDefinition: WebComponentDefinition = {
             attributes: [
                 {
                     name: "anchor",
+                    title: "Anchor",
                     type: DataType.string,
-                    description: "The anchor attribute",
+                    description:
+                        "The HTML id of the element the region is positioned relative to",
```

Comment:
```suggestion
                        "The HTML ID of the element the region is positioned relative to",
```

Reply:
I realize html attributes are lowercase, but when used in the description and not presented as `id` or referring directly to the attribute, I suggest following our convention with "HTML" and no using the rather uncommon word "id" instead.

Reply:
Updated all instances of this.